pkg/apis/eventsource/v1alpha1/openapi_generated.go

```diff
@@ -0,0 +1,4642 @@
+//go:build !ignore_autogenerated
```

Comment:
this file should not be here.

Reply:
@whynowy ack. The rebase was a little difficult. Will delete this file.